testing/agent-exporter/src/main/java/io/opentelemetry/javaagent/testing/exporter/AgentTestingExporterFactory.java

```diff
@@ -25,6 +28,15 @@
     return logExporter.getCollectedExportRequests();
   }
 
+  public static void forceFlush() {
+    // TODO(anuraaga): Flush metrics too.
+    List<CompletableResultCode> results =
+        Arrays.asList(
+            AgentTestingTracingCustomizer.spanProcessor.forceFlush(),
+            AgentTestingLogsCustomizer.logProcessor.forceFlush());
+    CompletableResultCode.ofAll(results).join(10, TimeUnit.SECONDS);
```

Comment:
WDYT about using `OpenTelemetrySdkAccess` instead? It flushes the meter provider too

Reply:
Oops realized that we don't want to flush metrics here anyways (there's no such thing as pending metric exports really)